src/frontend/src/handler/create_source.rs

```diff
@@ -697,6 +697,7 @@ pub(crate) fn bind_all_columns(
                 "Schema definition is required, either from SQL or schema registry.".to_string(),
             )));
         }
+        let non_generated_sql_defined_columns = non_generated_sql_columns(col_defs_from_sql);
```

Comment:
Is there a term for normally defined columns? 🤣
non-generated sounds really weird

Reply:
I hope there is  🤔